app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -72,7 +72,16 @@ export class ImageList extends React.Component<IServerGroupHeaderProps, IImageLi
   }
 
   public render() {
-    const images = this.props.images.sort();
+    const shorten = function(val: string): string {
+      const sha = val.lastIndexOf('sha256:');
+      if (sha > 0) {
```

Comment:
Since `lastIndexOf` will return -1 if no match is found, we should probably use `> -1` as the condition here, right?